src/components/elements/DropdownMenu.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { navStyleClasses } from 'styles/navbar';
+
+function DropdownMenu({ label, options }) {
```

Comment:
Abstracted the dropdown menu logic into a separate component so we can reuse for the other dropdown sections.